src/resolvers/raw.js

```diff
@@ -0,0 +1,36 @@
+'use strict'
+
+const errCode = require('err-code')
+const extractDataFromBlock = require('../utils/extract-data-from-block')
+const validateOffsetAndLength = require('../utils/validate-offset-and-length')
+
+const rawContent = (node) => {
+  return function * (options = {}) {
```

Comment:
I tend not to use this feature because it only works when `undefined` is passed and I usually also want to allow `null` to be passed (or any other falsey value really) so I end up doing the old `options = options || {}`. Kinda preference but also flexibility for the caller 🤷‍♀️